client/scss/components/_slider.scss

```diff
@@ -6,9 +6,14 @@
   // stylelint-disable declaration-no-important
   overflow: visible !important;
   // stylelint-enable declaration-no-important
+  will-change: transform;
+  transform: translateZ(0);
+  transition: transform 700ms ease;
 }
 
 .slider-controls {
+  will-change: opacity;
+  transform: translateZ(0);
   .control-arrow {
```

Comment:
I thought we had a lint rule to enforce a space before a nested block. Evidently not. Should we? Personally I think it scans easier with a space, but I'm not overly fussed.